File: API/Entities/Heel.cs
using System;

namespace API.Entities;

public class Heel
{
    public string Id { get; set; } = Guid.NewGuid().ToString();
    public required string Code { get; set; }
    public required int Price { get; set; }
    public required int Quantity { get; set; }
    public bool IsArchived { get; set; }
    public DateTime? ArchivedAt { get; set; }
}

File: CLIENT/src/types/Heel.ts
export interface Heel {
    id: string;
    code: string;
    price: number;
    quantity: number;
    isArchived: boolean,
    archivedAt: Date,
    images: HeelImage[];
}

export interface HeelImage {
    id: number;
    url: string;
    publicId: string;
    heelId: string;
    isMain: boolean;
}

